packages/atlas/src/hooks/useUniqueMemberHandle.ts

```diff
@@ -0,0 +1,58 @@
+import { useApolloClient } from '@apollo/client'
+import { useCallback } from 'react'
+
+import {
+  GetMembershipsDocument,
+  GetMembershipsQuery,
+  GetMembershipsQueryVariables,
+} from '@/api/queries/__generated__/memberships.generated'
+import { createId } from '@/utils/createId'
+
+function removeSpecialCharacters(input: string): string {
```

Comment:
my suggestion will be to have this as a util func in some utils file and rewrite it as `export const` just to maintain the  codestyle